### DISCUSSION TITLE
Project Update and a few near-term thoughts

### DISCUSSION THREAD

**Discussion by dannys42** [IBM-Swift/Kitura#1545] [General]
Hi all,

I'm still trying to figure out what method of communication is best, so I'm trying out this new GitHub feature.  I think we've updated most Kitura projects to reflect the change in org from `IBM-Swift` to `Kitura`.  If you notice anything off, please let me know.

There's of course more work to do:
* I'm going to start looking at the getting started guides and making sure they are up-to-date and working.  Along these lines I'm considering at converting the kitura CLI to Swift.  I'd like to also add some functionality there that to make things easier for maintainers working across the entire Kitura project.
* Along similar lines of keeping as much as possible Swift-based, I think it'd be interesting if someone wants to convert the documentation website from gatsby to `[Publish](https://github.com/JohnSundell/Publish)`
* I'm aware that the example projects still need to be migrated into Kitura.  I will probably also rename those projects so that they will be more easily discoverable in the repository list.  That said, the documentation should also have references to all th sample projects
* I have a base-line auth component using Auth0, if someone needs this, let me know.  But I'm focusing more on the getting started guides at this time.
* There are still CI issues in various projects.  In most cases it's not that CI is failing entirely.  We test against multiple versions of Swift and across macOS and Linux.  So sometimes it's just one or two versions that's failing for either OS.
* If you're using Kitura and want to share your experience, feel free to write up a blog post about it.
* Tutorials and example code are always welcome as well!

**Comment by mbarnach** [comment]
Hi @dannys42 

Good initiative!

For the blog suggestion, I think we should have some sort of reviewing committee, not to limit the contributions, but more to speed-up process. I think it could be very frustrating to write something, and not getting it published on time. If more people are reviewing/approving, we should be able to have a quick flow.
Personally, I would be interested to see the Auth0 you have made! Staying up to date with the industry is important.

Regards,
Mathieu
